packages/angular-cli/models/webpack-build-test.js

```diff
@@ -66,7 +67,7 @@ const getWebpackTestConfig = function (projectRoot, environment, appConfig) {
         { test: /\.(jpg|png)$/, loader: 'url-loader?limit=128000' },
         { test: /\.html$/, loader: 'raw-loader', exclude: [path.resolve(appRoot, appConfig.index)] }
       ],
-      postLoaders: [
+      postLoaders: debug ? [] : [
```

Comment:
In the latest webpack update, the loader configs changed a bit and is now a single `rules` array. 

You're probably going to have to add the `sourcemap-istanbul-instrumenter-loader` conditionally, but by itself, at the end.